aragon/aragon-react-boilerplate
# Updated react boilerplate for use with buidler

Fix https://github.com/aragon/aragon-cli/issues/1259

#### Changes
Also makes the boilerplate as vanilla as possible, removing things that may be overkill, like:
* linting
* coverage

And some things that are not yet supported by the Buidler plugin:
* templates
* publishing

### To test
1) Just check out the "buidler" branch in aragon-react-boilerplate (https://github.com/aragon/aragon-react-boilerplate/tree/buidler), 
2) `npm i`, 
3) then `npm start`, 
4) optionally `npm test`. 

Note 1: Since the plugin doesn't yet provide Aragon snapshot data, aragen must be started manually. You can do so by installing aragon-cli globally and running `aragon devchain start`.

Note 2: If the client in the browser doesn't show the app, you might need to wait a bit and refresh the tab manually. This is a known issue to be addressed in the plugin.
